.github/workflows/main.yml

```diff
@@ -1,11 +1,6 @@
 name: Continuous Integration
 
-on:
-  pull_request:
-    paths:

```

Comment:
I just noticed that the filter should also include the workflow folder as changes to those should be run as well I think... 🤔